modules/gis/src/lib/binary-to-geojson.d.ts

```diff
@@ -1,6 +1,10 @@
-import {BinaryGeometryData} from '../types';
+import {BinaryGeometryData, BinaryFeatureData} from '../types';
 
 /**
- * Binary to GeoJSON
+ * Convert binary geometry representation to GeoJSON
```

Comment:
You could specify what strings are valid. `'type: 'binary' | 'geojson` is valid typescript

Reply:
I got some `tsc` linting errors from the test files when I tried this